clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -430,10 +413,11 @@ public boolean mkdirs(Path cPath, FsPermission permission) throws IOException {
       mStatistics.incrementWriteOps(1);
     }
     TachyonURI path = new TachyonURI(Utils.getPathWithoutScheme(cPath));
-    MkdirOptions options =
-        new MkdirOptions.Builder(mTachyonConf).setRecursive(true).setAllowExists(true).build();
+    CreateDirectoryOptions options =
+        CreateDirectoryOptions.defaults().setRecursive(true).setAllowExists(true);
     try {
-      return mTFS.mkdir(path, options);
+      mTFS.createDirectory(path, options);
+      return true;
```

Comment:
Good point, I've updated the doc. Taking a quick look at the hadoop distributed fs implementation, it doesn't seem like false is ever returned (exception is always thrown on failure).